### ISSUE TITLE
How to connect Chrome with device

### ISSUE THREAD

**Issue by screendriver** [driftyco/ionic-box#23]
I can compile and deploy my ionic app to my USB connected device. But how can I connect and debug my app now with Chrome running on my host machine? Because the USB connection is exclusively given to VirtualBox running vagrant my Chrome cannot find my device. He only find it when I halt my Vagrant machine.
How did others solve this?

**Comment by AgDude** [comment]
Could the vagrant box somehow trick chrome into into thinking it is a device, and forward traffic to/from the actual device?

**Comment by Ionitron** [comment]
Greetings @screendriver!

I've closed this issue because my sensors indicated it was old and inactive, and may have already been fixed in recent versions of Ionic. However, if you are still experiencing this issue, please feel free to reopen this issue by creating a new one, and include any examples and other necessary information, so that we can look into it further.

Thank you for allowing me to assist you.

**Comment by screendriver** [comment]
Hi @Ionitron,

at the moment I do not use Vagrant anymore. I will reopen a new issue if I use it in the future.

**Comment by AgDude** [comment]
I managed to get this working. I ending up needing to install adb on the windows host, but it works well once it is all set up.

* Install adb on the windows host, and make sure the adb server is running
* Make sure you don't have any usb filters for yous VM (so the device will stay attached to windows). check in the virtualbox management window, the default ionic box creates new ports every time it starts.
* Change the init script in the Vagrantfile to make sure the adb server is not running on the guest `sudo /home/vagrant/android-sdk-linux/platform-tools/adb kill-server'
* Create a reverse ssh tunnel from windows to the guest. This can't be done in the Vagrantfile, those port forward forward a linux port to windows. This forwards a windows port to linux. I use openssh on windows, which lets me open the port with this command: `ssh -R 5037:localhost:5037 192.168.1.190 -l vagrant', but there are other tools which are easier to set up on windows and can accomplish the same thing.

Now you can use the adb command line tools from either windows or the vagrant box. And you can use chrome or gapdebug on windows.

**Comment by AbrahamArun** [comment]
@AgDude I am having the very same issue, but I am not using ionic box. Anyway, I have made sure that I have removed the USB filters from the vagrant config and that the adb server is killed in both the host and the guest machine. 

`ssh -R 5037:localhost:5037 192.168.50.4 -l vagrant`

`192.168.50.4` is the vm ip-address (OS X box). After doing the above:

when I run `adb devices`, I get the following error:

![screen shot 2016-02-15 at 6 08 36 pm](https://cloud.githubusercontent.com/assets/5391059/13048907/2f393704-d40f-11e5-85bc-e630760ca671.png)

Any idea what could be wrong?
